### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by Jobhdez**
## Motivation and Context
Josue looked around and clicked on links and determined that the FAQ link was the only one broken.

I also looked around for any broken links and I only found the FAQ link.

## How Has This Been Tested?

I did `yarn start` and everything worked as expected.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,4 +39,7 @@ yarn-error.log*
 
 # Emacs
 *~
-*#
\ No newline at end of file
+*#
+
+# firebase
+/firebase-export-1626142308993K36cpU/
```
Comment (josueetcom) [inline .gitignore:45]: A couple things:
1. Where does this come from? Do you know which command created this folder?
2. It looks like this has a generated ID. This rule will not work if similarly created `/firebase-export-*/` folders are made.
3. There's already a Firebase section in this file. This belongs there if anything.

**Comment by josueetcom** [comment]
One more comment. When you name PRs please make the title descriptive, having just an issue number there means more redirecting for whoever is trying to understand why the PR was merged.

**Comment by josueetcom** [comment]
Hey @Jobhdez, when are you going to update this to have the changes in #565 so this can get merged?

**Comment by Jobhdez** [comment]
> Hey @Jobhdez, when are you going to update this to have the changes in #565 so this can get merged?

when I tried to change the name of the branch it created a new branch and I merged that. This one should be closed?

**Comment by josueetcom** [comment]
Whoops nvm, it looks like you already merged #565.

**Pull request closed without merging**